src/services/update.js

```diff
@@ -0,0 +1,30 @@
+const sortTodo = (todos) => todos.sort((a, b) => a.index - b.index);
+export const save = (todos) => localStorage.setItem('todos', JSON.stringify(todos));
+
+export const saveToStorage = (todos) => {
+  const sortedTodos = sortTodo(todos);
+  save(sortedTodos);
+};
```

Comment:
- [ ] Your method for storing the array of object is good, but you still more to make it work.

![jdjdjd](https://user-images.githubusercontent.com/61562626/134658759-73d064ce-9358-475a-ba31-a0410ed30f42.png)